src/gm-apclientpp.cpp

```diff
@@ -615,3 +630,51 @@ double apclient_location_scouts(const char* locations_str, double create_as_hint
 
     return GM_BOOL(apclient && apclient->LocationScouts(locations_j, (int)create_as_hint));
 }
+
+double apclient_bounce(const char* data_str)
+{
+    if (api < 2) return GM_FALSE;
```

Comment:
In most coding styles, the body should start on the same line. (I try to stick to webkit c++ coding style now).
```suggestion
    if (api < 2)
        return GM_FALSE;
```